docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -551,7 +552,9 @@ There are two server-side files that need to be modified.
     * The MSGraphHelper class is defined in src\msgraph-helper.ts. 
     * We minimize the data that must be returned by specifying that we only want the name property and only the first 3 items.
 
-    `const graphData = await MSGraphHelper.getGraphData(graphToken, "/me/drive/root/children", "?$select=name&$top=3");`
+    ```javascript
+    const graphData = await MSGraphHelper.getGraphData(graphToken, "/me/drive/root/children", "?$select=name&$top=3");
```

Comment:
With `const` and `await`, isn't this TypeScript?

Reply:
Yes. This is a TypeScript walkthrough.

Reply:
Then the code type should be "typescript" instead of "javascript" :-)